packages/backend/src/accounting/transfers.ts

```diff
@@ -43,7 +39,16 @@ export async function createTransfers(
     }
     let flags = 0
     if (transfer.timeout) {
-      flags |= TransferFlags.two_phase_commit
+      flags |= TransferFlags.pending
+    }
+    if (transfer.pendingId && commit === true) {
+      flags |= TransferFlags.post_pending_transfer
+      transfer.id = uuid()
+    } else if (transfer.pendingId && commit === false) {
+      flags |= TransferFlags.void_pending_transfer
+      transfer.id = uuid()
+    } else {
+      transfer.pendingId = 0n
```

Comment:
I'd prefer to error for invalid transfer options than change them.

Reply:
Refer to: https://github.com/interledger/rafiki/issues/518